jerry-ext/include/jerryscript-ext/module.h

```diff
@@ -59,31 +59,109 @@ typedef struct jerryx_native_module_t
 #define JERRYX_NATIVE_MODULE(module_name, on_resolve_cb)  \
   JERRYX_NATIVE_MODULE_IMPLEM(module_name, on_resolve_cb)
 
-#define JERRYX_NATIVE_MODULE_IMPLEM(module_name, on_resolve_cb)        \
-  static jerryx_native_module_t _ ## module_name ## _definition =      \
```

Comment:
`static` here needs to be passed in as a macro parameter, because `extern` will not be able to find a `static` across two compilation units. Then, if you want to declare a module reachable from another compilation unit, you must call `JERRYX_NATIVE_MODULE_IMPLEM()` without `static`.